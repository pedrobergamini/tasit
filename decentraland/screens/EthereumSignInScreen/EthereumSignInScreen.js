import React from "react";
import EthereumSignIn from "@presentational/EthereumSignIn";

export default class EthereumSignInScreen extends React.Component {
  render() {
    const onConnect = () => {};
    return <EthereumSignIn onConnect={onConnect} />;
  }
}
