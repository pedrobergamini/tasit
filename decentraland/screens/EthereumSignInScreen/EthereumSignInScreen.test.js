import React from "react";
import NavigationTestUtils from "react-navigation/NavigationTestUtils";
import { shallow } from "enzyme";
import EthereumSignInScreen from "./EthereumSignInScreen";

describe("EthereumSignIn", () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it("renders the component", async () => {
    expect(shallow(<EthereumSignInScreen />)).toMatchSnapshot();
  });
});
