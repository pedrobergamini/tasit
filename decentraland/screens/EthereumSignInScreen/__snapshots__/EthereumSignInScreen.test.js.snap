// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EthereumSignIn renders the component 1`] = `
<EthereumSignIn
  onConnect={[Function]}
/>
`;
